clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -601,6 +601,60 @@ def CIR_VTableAttr : CIR_Attr<"VTable", "vtable", [TypedAttrInterface]> {
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// DynamicCastInfoAttr
+//===----------------------------------------------------------------------===//
+
+def CIR_DynamicCastInfoAttr : CIR_Attr<"DynamicCastInfo", "dyn_cast_info"> {
+  let summary = "ABI specific information about a dynamic cast";
+  let description = [{
+    Provide ABI specific information about a dynamic cast operation.
+
+    The `srcRtti` and the `destRtti` parameters give the RTTI of the source
```

Comment:
probably a good time to talk convention, for param names, do we want `srcRtti` or `src_rtti`? I have no strong preference, but we should try to be uniform (and encode expectations in some of our docs)

Reply:
It would be good to settle on one consistent style. A quick regex search of the source code leaves me with the impression that both CIR and MLIR dialects in general are pretty evenly split as to which style they use. I don't know if that reflects a transition over time or if it's just an accident. I see that a couple of dialects have declared for snake_case (which gets transformed to camelCase during code generation).